data/ecosystems/a/avalanche.toml

```diff
@@ -487,8 +488,8 @@ url = "https://github.com/ava-labs/avalanche-testing"
 url = "https://github.com/ava-labs/avalanche-types-rs"
 
 [[repo]]
-url = "https://github.com/ava-labs/avalanche-wallet"
 tags = ["Protocol"]
+url = "https://github.com/ava-labs/avalanche-wallet"
```

Comment:
can you keep the tags parameter below the URL like it was originally? not sure why it was changed